File: Search_in_mass.cpp
#include <iostream>
#include <list>

using namespace std;

int main(){
	int mass[9] = {1,43,34,875,956,2,46,0,64};
	cout << "Input number search" << endl;
	int a;
	cin >> a;
	for (int i = 0; i < sizeof(mass); i++){
		if (a == mass[i]){
			cout << i;
			break;
		}
	}
}

File: Search_in_mass.py
from array import *
x = int(input('Введите число которое хотели бы найти: '))
my_array = array('i', [1,32,525,363,142,2,313,8,543]) #Условимся что интовые значения
for i in my_array:
	if i == x:
		print('Индекс числа = ' + str(my_array.index(x)))
		break